packages/divider/test/benchmark/basic-test.ts

```diff
@@ -9,10 +9,11 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import '@spectrum-web-components/actionbar/sp-actionbar.js';
+
+import '@spectrum-web-components/divider/sp-divider.js';
 import { html } from '@spectrum-web-components/base';
 import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
 
 measureFixtureCreation(html`
-    <sp-actionbar open></sp-actionbar>
+    <sp-divider open></sp-divider>
```

Comment:
This `open` attribute is not needed here, and likely not on any of the other converted element except the Action Bar. Might be worth taking out of the generator.

Reply:
I have taken it out of the generator file.